src/main/java/seedu/address/model/order/Order.java

```diff
@@ -160,7 +170,7 @@ public String toString() {
             .append("; Order Date: ")
             .append(getOrderDate())
             .append("; Completed Date: ")
-            .append(getCompletedDate())
+            .append(getCompletedDate().get())
```

Comment:
use `.map(x -> x.toString()).orElse("-")` so that it does not throw null pointer exception

sorry forgot to update this in properly in previous commit

Reply:
Will update in the next commit